Update desktop tutorial

cheriannet opened the pull request:
Hi Roshan, 
 
I have updated the tutorial for Github Desktop.  Please take a look and merge it to the master.

Thanks,
Cherian.

cheriannet committed:
Merge pull request #1 from Roshanjossey/master

Update from Original

cheriannet committed:
Starting Modifications

Just added few assents and started updating the tutorial

cheriannet committed:
Changed asset downloaded

Updated the downloaded fork image.

cheriannet committed:
Added Status

Added status image

cheriannet committed:
Changed status code

Changed the code to show status image

cheriannet committed:
Create Branch

cheriannet committed:
Changes to Desktop Tutorial 01

In this I added few more assests and updated the desktop tutorial.md

cheriannet committed:
Changes to Desktop Tutorial 02

Added more edit to the desktop-tutorial.md

cheriannet committed:
Changes to Desktop Tutorial 03

I have added few more assets and then updated the desktop-tutorial.md.

cheriannet committed:
Changes to Desktop Tutorial 04

Fixed some spelling mistakes.

cheriannet committed:
Changes to Desktop Tutorial 05

Updated the README.md to add links to all the tutorials.

cheriannet commented:
How do I resolve the conflicts?

cheriannet committed:
Merge branch 'master' into Update-Desktop-Tutorial

cheriannet committed:
Merge branch 'master' into Update-Desktop-Tutorial

Roshanjossey reviewed (changes requested):
@cheriannet, I've added a few review comments. Please take a look at them.

Also, comments made on old tutorial applies for new one as well.

cheriannet committed:
Updated After Review by Roshan

Changed desktop-old-tutorial and desktop-tutorial to look same as the readme.md.  Links on top and bottom updated.

cheriannet commented on github-desktop-old-version-tutorial.md:
```diff
@@ -0,0 +1,126 @@
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=103)](https://github.com/ellerbrock/open-source-badges/)
+[<img align="right" src="https://firstcontributions.herokuapp.com/badge.svg">](https://firstcontributions.herokuapp.com)
```
Done.

cheriannet commented on github-desktop-old-version-tutorial.md:
```diff
@@ -0,0 +1,126 @@
+
+<img style="float: right;" src="assets/submit-pull.png" alt="submit pull request" />
+
+## Keeping your fork synced with this repository
```
Done.

cheriannet commented on github-desktop-old-version-tutorial.md:
```diff
@@ -0,0 +1,126 @@
+
+## Where to go from here?
+
+You could also join our slack team in case you need any help or have any questions. [Join slack team](https://firstcontributions.herokuapp.com)
```
Done.

cheriannet commented on README.md:
```diff
@@ -10,6 +10,10 @@ It's hard. It's always hard the first time you do something. Especially when you
 
 Reading articles & watching tutorials can help, but what's better than actually doing the stuff in a practice environment? This project aims at providing guidance & simplifying the way beginners make their first contribution. Remember: the more relaxed you are, the better you learn. If you are looking to make your first contribution, just follow the simple steps below.
 
+You can use Github on command line, [Github Desktop](github-desktop-tutorial.md) , [Visual Studio 2017](github-windows-vs2017-tutorial.md) or [Gitkraken](gitkraken-tutorial.md).
```
That's a good point but I added that link up there because when I did my first contributions, I didn't know there was a tutorial available for desktop version. So I fumbled with the command prompt version to figure-out how to do it with desktop. I couldn't find the link to desktop tutorial even after you told me it was available. Hope you remember that. So I think it is better to keep it up there. Let me know if you still want me to remove it. Thanks! :)

Roshanjossey replied:
From the feedback we've collected so far, most people use command line. Also, most people find the desktop app tutorial easily. I think it's better to remove this.

cheriannet replied:
Ok. I have removed the link and updated the repository.  Thank you for your feed back.  If you want me to update anything else, Let me know.

cheriannet commented on github-desktop-tutorial.md:
```diff
@@ -65,43 +73,66 @@ Replace `<your-name>` with your name
-## Keeping your fork synced with this repository
+You have now successfully completed your first github source code update. Please give yourself a pat in the back. :) 
+
+## Keeping your fork synced with this repository (Optional)
```
Done.

cheriannet committed:
Delete Assests not used

Deleted the assets not used in the project because we removed the sync section on the desktop tutorial.

cheriannet committed:
Removed link to desktop totorial from readme

Roshanjossey commented on github-desktop-old-version-tutorial.md:
```diff
@@ -0,0 +1,133 @@
+
+<a href="http://saasgrids.com"> <img alt="http://saasgrids.com" src="assets/saasgrids-banner.png" width="500"></a>
+
+Here's some beginner level issues in popular repos that you can solve. Go ahead and go to those repos to learn more
```
@cheriannet, could you please remove this section as well? We don't need this as this is in our web app now. Also remove this from new version as well.

cheriannet replied:
Done. Also removed links from the Visual Studio Tutorial and GitKraken Tutorial.

cheriannet committed:
Remved Popular Repos link from the bottom

Removed the link to all the popular repors from the bottom because it is already in the apps section. Updated desktop old and new. Also did the same for visual studio tutorial.

cheriannet committed:
Updated gitkraken tutorial

Updated the top and the bottom links of gitkraken tutorial. Also removed the sync portion from the tutorial.

Roshanjossey commented:
Thank you for your time and effort you spend on this @cheriannet.

Merging this.

cheriannet commented:
No problem. It was fun and a huge learning experience for me. Thank you for creating and maintaining this project in Github.
